app/views/coronavirus/timeline_entries/_form.html.erb

```diff
@@ -11,7 +11,7 @@
 
 <%= render "components/markdown_editor", {
   label: {
-    text: "Content",
+    text: t("coronavirus.pages.timeline_entries.form.content"),
```

Comment:
I think this and the previous key would benefit from having a suffix of `_label` I thought they were heading and content of the form rather than references to form field labels.